corehq/apps/mocha/templates/mocha/base.html

```diff
@@ -64,4 +64,7 @@
       }
     </script>
   </body>
+  {% if show_status_page %}
```

Comment:
i don't think we need this for tests. I would leave the status page off of `mocha/base.html` as it is better to avoid loading external scripts in the test template.